src/Pages/Profile/profile.scss

```diff
@@ -30,3 +30,29 @@
     }
   }
 }
+
+.profile-missions {
+  width: 48%;
+  display: flex;
+  flex-direction: column;
+
+  & > h2 {
+    margin-bottom: 1rem;
+    font-weight: 400;
+  }
+
+  & > .missions-joined {
+    border-radius: 4px;
+    border: 1px solid rgb(192 192 192);
+
+    & > .missionName {
+      padding: 2rem;
+      width: 100%;
+      border-bottom: 1px solid rgb(192 192 192);
+    }
+
+    & > :last-child {
+      border-bottom: none;
+    }
+  }
+}

```

Comment:
[Optional] Seems like profile-rockets and profile-missions have the same style properties. It would be better to just add mission classes next to rocket classes to avoid code repetition.

```CSS
.profile-rockets,
.profile-missions {
  width: 48%;
  display: flex;
  flex-direction: column;

  & > h2 {
    margin-bottom: 1rem;
    font-weight: 400;
  }

  & > .reserved-rockets,
  .missions-joined {
    border: 1px solid rgb(192 192 192);
    border-radius: 4px;

    & > p,
    .missionName {
      padding: 2rem;
      width: 100%;
      border-bottom: 1px solid rgb(192 192 192);

      &:last-child {
        border-bottom: none;
      }
    }
  }
}
```